src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
Header should have full page width on higher resolutions, don't hardcode these values.